src/App.tsx

```diff
@@ -1,14 +1,60 @@
-/* eslint-disable max-len */
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Status } from './types/Status';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
+export const filteredQueryInput = (
+  comparedTodos: Todo[],
+  query: string,
+): Todo[] => {
+  if (query) {
+    return comparedTodos.filter(todo =>
+      todo.title.toLowerCase().includes(query.toLowerCase()),
+    );
+  }
+
+  return comparedTodos;
+};
+
+const helperStatusButton = (filter: Status, todos: Todo[]): Todo[] => {
+  const statusFilter = todos;
+
+  switch (filter) {
```

Comment:
```suggestion
const filterTodosByStatus = (filter: Status, todos: Todo[]): Todo[] => {
  switch (filter) {
```
It doesn't look like a button, it's a function